compiler/rustc_index/src/bit_set.rs

```diff
@@ -209,7 +209,7 @@ impl<T: Idx> BitSet<T> {
             self.words[start_word_index] |= !(start_mask - 1);
             // And all trailing bits (i.e. from 0..=end) in the end word,
             // including the end.
-            self.words[end_word_index] |= end_mask | end_mask - 1;
+            self.words[end_word_index] |= end_mask | (end_mask - 1);
```

Comment:
lol was this a bug? or is the precedence alright implicitly like those parens?

Reply:
according to clippy the precedence was already like this but since its no obvious it suggested the parens :upside_down_face:

Reply:
Makes sense.